### DISCUSSION TITLE
Add ability to warm-up the website

### DISCUSSION THREAD

**Discussion by stefankip** [umbraco/Umbraco.Cloud.Issues#784] [Ideas]
When deploying changes which require a restart of the application pool, it would be nice if the website got warmed up, so the first request doesn't take that long.

**Comment by sitereactor** [comment]
Hey @stefankip this is something that has been on our internal wishlist for some time. But haven't been able to do so due to limitations in our infrastructure. We are currently in the process of modenizing our cloud setup, so this should hopefully be one of the first features we'll add once we are "done".

I'm afraid I don't have a timeframe for when it will be available, but I'd like to stress that this is something we want as well!

**Comment by kipusoep** [comment]
Great to hear! I just dropped the idea here so it would be out in the open, no hurries :-)
